### PR TITLE
Change output of gravity lines containing numbers.

### PR DESCRIPTION
**Pull request by PromoFaux**
**By submitting this pull request, I confirm the following:** `{please fill any appropriate checkboxes, e.g: [X]}`

`{Please ensure that your pull request is for the 'development' branch!}`

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template.
- [x] I have made only one major change in my proposed changes.
- [x] I have commented my proposed changes within the code.
- [x] I have tested my proposed changes, and have included unit tests where possible.
- [x] I am willing to help maintain this change if there are issues with it later.
- [x] I give this submission freely and claim no ownership.
- [x] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [x] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))
- [x] I have Signed Off all commits. ([`git commit --signoff`](https://git-scm.com/docs/git-commit#git-commit---signoff))

---

**What does this PR aim to accomplish?:**

Changes the output of lines in gravity

**How does this PR accomplish the above?:**

Grammar is changed on lines where there are numbers

**What documentation changes (if any) are needed to support this PR?:**

None

### CODE REVIEW

**Comment by PromoFaux** [comment]
image

File: gravity.sh [hunk 1]
```diff
@@ -415,20 +414,19 @@ gravity_Filter() {
...
   # Format $preEventHorizon line total as currency
   num=$(printf "%'.0f" "$(wc -l < "${piholeDir}/${preEventHorizon}")")
-  echo -e "  ${INFO} ${COL_BLUE}${num}${COL_NC} unique domains trapped in the Event Horizon"
+  echo -e "  ${INFO} Number of unique domains trapped in the Event Horizon: ${COL_BLUE}${num}${COL_NC}"
 }
 
 # Whitelist unique blocklist domain sources
 gravity_WhitelistBLD() {
-  local uniqDomains plural="" str
+  local uniqDomains
```
Comment (DL6ER) [inline gravity.sh]: `str` not `local` anymore although used 7 lines below?

File: gravity.sh [hunk 2]
```diff
@@ -415,20 +414,19 @@ gravity_Filter() {
...
 
   echo ""
 
   # Create array of unique $sourceDomains
   mapfile -t uniqDomains <<< "$(awk '{ if(!a[$1]++) { print $1 } }' <<< "$(printf '%s\n' "${sourceDomains[@]}")")"
-  [[ "${#uniqDomains[@]}" -ne 1 ]] && plural="s"
 
-  str="Adding ${#uniqDomains[@]} blocklist source domain${plural} to the whitelist"
+  str="Number of blocklist source domains being added to the whitelist: ${#uniqDomains[@]}"
```
Comment (dschaper) [inline gravity.sh]: Needs to be declared `local` as per DL's comment.

File: gravity.sh (hunk 1 above)
Comment (PromoFaux) [reply gravity.sh]: Oh yeah, accidental deletion on that one! Will add it back in

**Pull request merged**